src/axom/quest/examples/quest_distributed_distance_query_example.cpp

```diff
@@ -191,7 +207,7 @@ struct BlueprintParticleMesh
   /// Returns true if points have been added to the particle mesh
   bool hasPoints() const
   {
-    return m_coordsGroup != nullptr && m_coordsGroup->hasGroup("values");
+    return m_coordsGroup != nullptr && m_coordsGroup->hasView("values/x");
```

Comment:
Would the `hasGroup` and `hasView` calls as coded ever return different values?  Just curious about why the change.  I was assuming that correct blueprint data means that if "values" exist, so does "values/x".

Reply:
It was mostly because I was actually grabbing `"values/x"` a few lines below, so it's a bit more of a safeguard.